src/ontology/uberon-edit.obo

```diff
@@ -102924,8 +102926,8 @@ xref: FMA:7223
 xref: RETIRED_EHDAA2:0000884
 xref: SCTID:277708006
 xref: VHOG:0000790
-is_a: UBERON:0004111 ! anatomical conduit
 is_a: UBERON:0006846 {source="EHDAA2"} ! surface groove
+is_a: UBERON:0013686 ! anatomical conduit space
```

Comment:
I think "interventricular sulcus" should be an exact, not related, synonym and should be modelled like other sulci or grooves instead of being classified as an ['anatomical conduit space'](http://purl.obolibrary.org/obo/UBERON_0013686) as I don't believe a lumen is present.

Reply:
ah thats interesting, I guess previous modeling was wrong too then, I think just removing this axiom should solve it :)

Reply:
> I think "interventricular sulcus" should be an exact, not related, synonym.

Would you mind helping me do a separate PR/ticket for this please? (sorry dealing with this branch atm and all) - would prefer this to be separated from this PR :)

Reply:
PR in #2704